examples/byo-model-outputs.ipynb

```diff
@@ -5,76 +5,62 @@
    "id": "a8dae933-0b5f-4c6f-b9fc-145e467071b9",
```

Comment:
<ul><li><code>In <s>other</s> use cases that we <strong>demonstrate</strong> in the other example notebooks</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/fmeval/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>